crates/re_renderer/src/renderer/rectangles.rs

```diff
@@ -653,3 +677,90 @@ impl Renderer for RectangleRenderer {
         ]
     }
 }
+
+fn is_float_filterable(format: &wgpu::TextureFormat) -> bool {
+    // based on https://gpuweb.github.io/gpuweb/#texture-format-caps,
+    // TODO(emilk): add this to the wgpu crate
```

Comment:
This information is already available via https://github.com/gfx-rs/wgpu/blob/v0.15/wgpu-types/src/lib.rs#L1754
(i.e.  TextureFormat::describe().flags.contains(wgpu::TextureFormatFeatureFlags::FILTERABLE)`)


Note that in latest wgpu this isn't as hidden, as the texture format features got refactored there - `guaranteed_format_features`
https://github.com/gfx-rs/wgpu/blob/trunk/wgpu-types/src/lib.rs#L2706)